packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -376,7 +379,9 @@ export class PayPalClient {
     if (resultObj.ACK === 'Success' || resultObj.ACK === 'SuccessWithWarning') {
       return resultObj;
     } else {
-      throw new PayPalClientError(result.text, resultObj);
+      const err = new PayPalClientError(result.text, resultObj);
+      this.log.error('paypalClient.doRequest', err);
+      throw err;
```

Comment:
We could add a log here, but IIUC those alone would only show up in BigQuery not Sentry, so we probably want to also modify the `PayPalClientError`, as its `this.message` is what we're seeing in Sentry right now.

If you look at that class' `constructor` method, you'll see that we parse the NVP response (initially `this.raw`, a string, for the `PayPalClientError` instance) into an object with type `NVPResponse` (`this.data`).

I think we want to make sure that one of those two values (`this.data` would be easier to read) is sent as part of the `PayPalClientError` to Sentry.

@bbangert or @chenba , is there a good example of adding this extra key-value data to an error captured by Sentry? Looking at [`AppError`](https://github.com/mozilla/fxa/blob/9838b2df66fcbaf2014b7b210e44d16a34439511/packages/fxa-auth-server/lib/error.js#L175), I see we have a `this.output.payload` which may be the right place?

Reply:
Answer: `Sentry.withScope`